src/esp/physics/RigidObject.h

```diff
@@ -80,6 +81,47 @@ enum class RigidObjectType {
 
 };
 
+/**@brief Convenience class for applying constant velocity control to a rigid
+ * body. */
+class VelocityControl {
+ public:
+  /**@brief Constant linear velocity. */
+  Magnum::Vector3 linVel;
```

Comment:
linVel_

Reply:
Those are all public members, isn't the guideline to have only privates underscored?

Reply:
Well, this is a good point here. Good catch.

My understanding is that by default all the variables in the class should not be public, and thus have the underscore.

Since I suggested using the struct, the variables here are public. That means the underscore is not needed.